src/components/organisms/settings/menu.tsx

```diff
@@ -3,19 +3,21 @@ import React from "react";
 import { useTranslation } from "react-i18next";
 import { matchPath, useLocation } from "react-router-dom";
 
-import { menuUserItems } from "@constants";
+import { userMenuItems, userMenuOrganizationItems } from "@constants";
 import { cn } from "@src/utilities";
 
 import { Button, IconSvg } from "@components/atoms";
 
-export const SettingsMenu = () => {
+export const SettingsMenu = ({ variant }: { variant: "userSettings" | "organization" }) => {
```

Comment:
Pass menu items array as prop, not variant

Reply:
discussed